uPortal-webapp/src/main/resources/properties/contexts/schedulerContext.xml

```diff
@@ -63,7 +63,7 @@
         <task:scheduled ref="portalEventDaoQueuingEventHandler" method="flush" fixed-delay="997"/> <!-- ~1 second period -->
         <task:scheduled ref="cacheManagerExpiredElementEvictor" method="evictExpiredElements" fixed-delay="61900"/> <!-- ~1 minute period -->
         <task:scheduled ref="cacheHealthReporterService" method="generateReports" fixed-delay="60013"/> <!-- ~1 minute period -->
-        <task:scheduled ref="portalSearchIndexer" method="updateIndex" fixed-delay="${org.apereo.portal.index.portalSearchIndexer.updateIndexPeriod:180001}"/> <!-- ~3 minute period -->
+        <task:scheduled ref="portalSearchIndexer" method="updateIndex" fixed-delay="${org.apereo.portal.index.portalSearchIndexer.updateIndexPeriod:58001}"/> <!-- ~1 minute period -->
```

Comment:
Would this have a significant affect on performance?
Or is indexing trivial in this case?

Reply:
Good question!
Performance seems reasonable. This is replacing the calls doing similar work for each search request. It's also configurable for those institutions that see issues.

Reply:
And we won't see affects if not enabled :laughing: (sorry hard weeks due to confinement and so school apps were never used as much but all is running well after 2 days of works :eyeglasses:  !)

An idea, for all these task we should add a doc that explain how to watch on them if there is a cost on server ressources

Reply:
Agree that we need to document parameters to consider when addressing performance.